draw.typ

```diff
@@ -256,7 +265,7 @@
   ct,
   angle: 0deg,
   anchor: none,
-  padding: 0.5em,
```

Comment:
This should be `auto`. And if `auto` then use ctx-padding, otherwise use parameter.

Reply:
Okay, the latest commit sets it to auto. Thanks :)